.devcontainer/vmr-source-build/README.md

```diff
@@ -13,7 +13,7 @@ This Codespace can help you debug the source build of .NET. In case you have run
 `/workspaces/dotnet` with the PR changes pulled into it. You can then attempt to source-build
 the VMR which is what the VMR leg in the installer PR build doing. This build takes about 45
 minutes and, after completion, produces an archived .NET SDK located in
-`/workspaces/dotnet/artifacts/x64/Release`.
+`/workspaces/dotnet/artifacts/packages/Release/Shipping`.
```

Comment:
The VMR's readme needs a similar change in step 2 and 3 of [Building](https://github.com/dotnet/dotnet?tab=readme-ov-file#building)

Reply:
I missed that you already updated it.